app/models/event.rb

```diff
@@ -1,3 +1,14 @@
 class Event < ApplicationRecord
-  belongs_to :organizer
+  scope :active, -> { where cancelled: false }
+
+  belongs_to :organizer, class_name: 'User', foreign_key: 'organizer_id'
+  has_many :registrations
+  has_many :attendees, through: :registrations, source: :user
+
+  validates :name, presence: true, length: { in: 6..20 }
+  validates :description, presence: true, length: { maximum: 500 }
```

Comment:
text in PG already has max character limit of 255. Please check once and remove accordingly